model/sys/pma.sail

```diff
@@ -116,6 +116,18 @@ function matching_pma(pmas : list(PMA_Region), addr : physaddr, width : mem_acce
   }
 }
 
+// Get the first PMA region that fully contains a given [base, base+size) range.
+function matching_pma_range(pmas : list(PMA_Region), base : bits(64), size : bits(64)) -> option(PMA_Region) = {
```

Comment:
Maybe should be called `matching_pma_bits` seeing as the distinction is that the arguments are bits instead of integers?